### PR TITLE
Replace conda packages of moose dependencies

### PR DESCRIPTION
**Pull request by smpark7**
Replaces instructions for installing `moose-tools` and `moose-libmesh` conda packages with `moose-dev` as is the latest practice for MOOSE apps.

This PR also fixes the [failing gh-pages deployment](https://github.com/arfc/moltres/actions/runs/7390876488/job/20106659549) as a result of the missing wasp dependency (`moose-dev` contains wasp).

### CODE REVIEW

**Review by LukeSeifert — APPROVED** [review]
Looks good to me! @ me if you'd like me to go ahead and merge, otherwise I can wait for another review if you'd like.

**Comment by smpark7** [comment]
Thanks @LukeSeifert! Yea this is good to merge

**Pull request merged**
